github/repos_community_health.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+	"time"
+)
+
+// CommunityHealthMetrics represents a response containing the community metrics of a repository.
+type CommunityHealthMetrics struct {
+	HealthPercentage *int `json:"health_percentage"`
+	Files            struct {
+		CodeOfConduct struct {
+			Name    *string `json:"name"`
+			Key     *string `json:"key"`
+			URL     *string `json:"url"`
+			HTMLURL *string `json:"html_url"`
+		} `json:"code_of_conduct"`
+		Contributing struct {
+			URL     *string `json:"url"`
+			HTMLURL *string `json:"html_url"`
+		} `json:"contributing"`
+		License struct {
+			Name    *string `json:"name"`
+			Key     *string `json:"key"`
+			URL     *string `json:"url"`
+			HTMLURL *string `json:"html_url"`
+		} `json:"license"`
+		Readme struct {
+			URL     *string `json:"url"`
+			HTMLURL *string `json:"html_url"`
+		} `json:"readme"`
+	} `json:"files"`
+	UpdatedAt time.Time `json:"updated_at"`
```

Comment:
Should this be a pointer too, for consistency? Other structs have it as pointer.